footprints/main/models.py

```diff
@@ -994,8 +969,15 @@ def display_title(self):
             return self.title
 
     def owners(self):
-        role = Role.objects.get_owner_role()
-        return self.actor.filter(role=role)
+        return self.actor.filter(
+            role__name=Role.OWNER).select_related('person')
+
+    def owners_list(self):
+        a = ''
+        for actor in self.actor.all():
+            if actor.role.name == Role.OWNER:
+                a += actor.display_name() + ', '
+        return a[:-2]
```

Comment:
Then I'd say, just use the direct `self.actor.filter(role__name=Role.OWNER)`, instead of `self.owners()`. I'd also say that if `.owners()` can't be called from just anywhere in the code without dire performance consequences, that deserves a big warning label somewhere.